app/src/main/kotlin/com/hedvig/app/feature/chat/ui/ChatActivity.kt

```diff
@@ -147,11 +144,7 @@ class ChatActivity : AppCompatActivity(R.layout.activity_chat) {
         handleSingleSelectLink(
           value = value,
           onLinkHandleFailure = {
-            authenticationTokenService.authenticationToken = null
-            lifecycleScope.launch {
-              featureManager.invalidateExperiments()
-            }
-            startActivity(MarketingActivity.newInstance(this, true))
```

Comment:
Another case where this should be handled by the to come PR where activities observe the token to do this automatically.